src/coreclr/jit/compiler.cpp

```diff
@@ -2057,18 +2057,36 @@ unsigned Compiler::compGetTypeSize(CorInfoType cit, CORINFO_CLASS_HANDLE clsHnd)
 
 CorInfoCallConvExtension Compiler::compMethodInfoGetEntrypointCallConv(CORINFO_METHOD_INFO* mthInfo)
 {
-    CorInfoCallConv callConv = mthInfo->args.getCallConv();
-    if (callConv == CORINFO_CALLCONV_DEFAULT || callConv == CORINFO_CALLCONV_VARARG)
+    switch (mthInfo->args.getCallConv())
```

Comment:
I think it would be better to just call `info.compCompHnd->getUnmanagedCallConv` here and the EE side deal with  the different calling convention metadata encodings.

It should allow us to delete `CORINFO_CALLCONV_C`, `CORINFO_CALLCONV_STDCALL`, etc. and have just one unmanaged calling convention enum on the JIT/EE interface.